draft-ietf-quic-transport.md

```diff
@@ -1207,41 +1208,53 @@ be routed back to the endpoint and identified by the endpoint upon receipt.
 
 A zero-length connection ID MAY be used when the connection ID is not needed for
 routing and the address/port tuple of packets is sufficient to associate them to
-a connection. An endpoint whose peer has selected a zero-length connection ID
+a connection.  An endpoint whose peer has selected a zero-length connection ID
 MUST continue to use a zero-length connection ID for the lifetime of the
 connection and MUST NOT send packets from any other local address.
 
-When an endpoint has requested a non-zero-length connection ID, it will issue a
-series of connection IDs over the lifetime of a connection. The series of
-connection IDs issued by an endpoint is ordered, with the final connection ID
-selected during the handshake coming first.  Additional connection IDs are
-provided using the NEW_CONNECTION_ID frame ({{frame-new-connection-id}}), each
-with a specified sequence number.  The series of connection IDs issued SHOULD be
-contiguous, but might not appear to be upon receipt due to reordering or loss.
-
-Each connection ID MUST be used on only one local address. When packets are sent
-for the first time on a new local address, a new connection ID MUST be used with
-a higher sequence number than any connection ID previously used on any local
-address.  At any time, an endpoint MAY change to a new connection ID on a local
-address already in use.
-
-An endpoint MUST NOT send packets with a connection ID which has a lower
-sequence number than the highest sequence number of any connection ID ever sent
-or received on that local address.  This ensures that when an endpoint migrates
+An endpoint whose peer has selected a non-zero-length connection ID will
+potentially use multiple connection IDs over the lifetime of a connection.
+These connection IDs are supplied by the peer using the NEW_CONNECTION_ID frame
+({{frame-new-connection-id}}).
+
+An endpoint considers the entire set of connection IDs that its peer has issued
+to be valid for use at any time. When the peer issues a new connection ID, it
+MUST accept packets using this connection ID for the duration of the connection
+or until the peer invalidates the connection ID via a CONNECTION_ID_FINISHED
```

Comment:
The use of peer gets confusing. Earlier in the paragraph, peer was the other guy who was issuing the connection IDs. Now its the one to whom they are issued. Perhaps use a sender/receiver relationship?

Reply:
Split this out into two sections